modules/cnf-creating-nrop-cr.adoc

```diff
@@ -73,4 +73,28 @@ pod/numaresources-controller-manager-7d9d84c58d-qk2mr   1/1     Running   0
 pod/numaresourcesoperator-worker-7d96r                  2/2     Running   0          97s
 pod/numaresourcesoperator-worker-crsht                  2/2     Running   0          97s
 pod/numaresourcesoperator-worker-jp9mw                  2/2     Running   0          97s
-----
\ No newline at end of file
+----
+
+If you need to configure NUMA-aware scheduling for multiple `MachineConfigPools`, you must define separate `NodeGroups`, each mapping to a single `MachineConfigPool`. For example, to configure NUMA-aware scheduling for `worker-cnf`, `worker-ht` and `worker-other` `MachineConfigPools`, you must define three separate NodeGroups in the `NUMAResourcesOperator` CR as shown in the following example:
```

Comment:
Consider swapping out `MachineConfigPools` to reduce monospaced nouns for "machine config pools (MCPs)".